src/psyclone/tests/utilities_test.py

```diff
@@ -149,7 +149,6 @@ def test_get_invoke():
     # First test all 5 valid APIs - we only make sure that no exception
```

Comment:
"..4 valid..." unless dynamo0.1 has gone in which case it's 3.

Reply:
Copyright date pls.

Reply:
Done